packages/@vuepress/theme-default/src/client/components/ToggleDarkButton.vue

```diff
@@ -49,12 +49,27 @@ const setDarkClass = (): void => {
   htmlEl?.classList.toggle('dark', isDark.value)
 }
 
+const updateThemeColor = (): void => {
+  const themeColorEl = window?.document.querySelector(
+    'meta[name="theme-color"]'
+  )
+  const newThemeColor = getComputedStyle(
```

Comment:
Should explicitly use `window.getComputedStyle`

Reply:
https://github.com/vuepress/vuepress-next/blob/b0a9815c9eeaa2ba623cc1a153139e85397d450e/packages/%40vuepress/theme-default/src/client/components/ToggleDarkButton.vue#L54

Got resolved, thanks.